FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -230,31 +247,53 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     Similar to the one in <bibl><ptr target="bm:Grohmann1919Marien"/><citedRange unit="page">30</citedRange></bibl> 
                                     and in <bibl><ptr target="bm:Budge1933Miracles"/><citedRange unit="page">6</citedRange></bibl></note>
                        </msItem> 
-                            <layoutDesc></layoutDesc>
+                        </msContents>
+                        <physDesc>
```

Comment:
`msContents `tag must be closed here
`physDesc `must start here

`layout `moved to `physDesc`